src/app/Shared/ClearSelectorButton.component.tsx

```diff
@@ -47,18 +47,21 @@ export function ClearSelectorComponent(props: {
     return (
         <div css={buttonAreaStyle}>
             <LightTooltip title={mouseOverText} placement="top">
-                <IconButton
-                    css={iconButtonStyle}
-                    onClick={handleClick}
-                    disabled={props.disabled}
-                    size="large">
-                    <CancelIcon
-                        css={css`
-                            height: 20px;
-                            width: 20px;
-                        `}
-                    />
-                </IconButton>
+                <span>
+                    <IconButton
+                        css={iconButtonStyle}
```

Comment:
iconButtonStyle: display: inline-block or so could have worked too.

Reply:
This is the official solution that Material Ui recommends.

Reply:
👍